ui-admin/src/study/notifications/NotificationContent.tsx

```diff
@@ -54,41 +56,48 @@ export default function NotificationContent({ studyEnvContext, portalContext }:
 
   return <div className="container-fluid px-4 py-2">
     { renderPageHeader('Participant email configuration') }
-    {isLoading && <LoadingSpinner/>}
-    {!isLoading && <div className="row">
-      <div className="col-md-3 mh-100 bg-white border-end">
-        <ul className="list-unstyled p-2">
-          { CONFIG_GROUPS.map(group => <li key={group.type}>
-            <h6 className="pt-2">{group.title}</h6>
-            <ul className="list-unstyled p-2">
-              { configList
-                .filter(config => config.notificationType === group.type)
-                .map(config => <li key={config.id} className="py-1">
-                  <div className="d-flex">
-                    <NavLink to={`configs/${config.id}`} style={navStyleFunc}>
-                      <NotificationConfigTypeDisplay config={config}/>
-                      <span className="text-muted fst-italic"> ({deliveryTypeDisplayMap[config.deliveryType]})</span>
-                    </NavLink>
-                  </div>
-                </li>
-                ) }
+    <div className="d-flex">
+      {isLoading && <LoadingSpinner/>}
+      {!isLoading && <div className="row">
+        <div className="col-md-3 mh-100 bg-white">
```

Comment:
same, I think you can get rid of the bootstrap row/col classes and divs above

Reply:
Done